src/test/java/com/coreos/jetcd/internal/impl/LoadBalancerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.coreos.jetcd.internal.impl;
+
+import com.coreos.jetcd.ClientBuilder;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.kv.PutResponse;
+import io.grpc.PickFirstBalancerFactory;
+import io.grpc.util.RoundRobinLoadBalancerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
```

Comment:
No I don't. 

I'm used to JUnit because is the most used one for the project I'm involved but I do not care too much.